.github/workflows/publish.yml

```diff
@@ -70,8 +70,28 @@ jobs:
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
           publish_dir: _build/html/
+          # cname: julia.quantecon.org
       - name: Upload "_build" folder (cache)
         uses: actions/upload-artifact@v2
         with:
           name: build-publish
           path: _build
+      # Sync Download Notebooks
+      - name: Prepare lecture-julia.notebooks sync
+        shell: bash
+        run: |
+          mkdir -p _build/lecture-julia.notebooks
+          cp -a _notebook_repo/. _build/lecture-julia.notebooks
```

Comment:
@Alim-faraji @jlperla @jbrightuniverse the base files for the `notebook` repo will need to be in `_notebook_repo/` so that it populates the `notebook` repo. These files need to be maintained here as the `push-to-another` repo will wipe them if they aren't synced each time.

Reply:
I added placeholders for this for the testing.  I think I will merge tonight.